packages/eslint-plugin/test/test-safe-await-separator.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-env node */
+
+'use strict';
+
+// Convenience utility using Mocha globals.
+const { RuleTester } = require('eslint');
+
+const ruleTester = new RuleTester({
+  parserOptions: { sourceType: 'module', ecmaVersion: 2021 },
+});
+const rule = require('../lib/rules/safe-await-separator.js');
+const {
+  validSafeAwaitSeparator,
+  subtlyValid,
+  clearlyInvalid,
+} = require('./corpus.js');
+
+const unsafeAwaitSeparator = 'unsafeAwaitSeparator';
```

Comment:
surprised to see this string defined as const. since this is a test I think it's appropriate to inline the literal. if you prefer not to, please do the same for `'unexpectedNestedAwait'`

Reply:
Vestiges of an earlier commit.  Done!